Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,462 @@
+This included the flow system and how it operates. Is it safe and is it necessary? Those questions were answered by Dr. Garrick of the MET department.
+There may be some assistance required if the nozzle performance if far below nominal, but guessing from past work it will not be a problem. There may also
+be the need for assistance during the data analysis phase or if the team chooses to perform flow simulations. But it will be a small amount of help.
+
+For the data analysis, if assistance is required for fully understanding it, Dr. Olles would be a good professor to ask. He has a PhD in rocket science
+so he could certainly help. Also, a SPEX member, Varun, has experience in flow simulation so he could probably understand the data sufficiently too.
+\subsection{Long-Term Vision}
+\label{sec:vision}
+
+The long term vision of this project is to be a building block to the next phase of space propulsion research. CREST, a mission that was theorized in the
+spring of 2016 by James Parkus, relies on a space propulsion system. It is the reason the SPEX propulsion subsystem was formed. Thus, it is natural for the
+CGT to assist in the furthering of this project as the complication of the propulsion system has been the main concern. The problem found with this was the required velocity change.
+The CubeSat would need to change its orbital velocity by 122 $ms^{-1}$, opposite its direction of motion. To do this with a cold gas thruster one needs a certain amount
+of propellant varing with the chamber pressure. If you increase the pressure, one can hold more propellant but the optimal value of both was far higher than we anticipated.
+With a spherical tank the chamber pressure would be near 15000 Psi and with a rectangular tank the pressure would be near 8000 Psi. Because this was the first propulsion system
+we had designed, we were doubtful that NASA would accept these conditions. Therefore, we are making one. This project will also be great for
+the members to put on their resume. At the end of this project a detailed document will be made which outlines the most important aspects of the project.
+Then the RCS will build off the momentum created by this project. And the understanding of space propulsion inside SPEX will continue to grow and flourish.
+
+Once the CGT is complete, the team can move onto the next phase of space propulsion research. This is the RCS\@. Reaction control systems are used very
+frequently in space propulsion and attitude adjustment systems as well as rockets. SpaceX uses them on the falcon 9 for landing orientation adjsutments. You
+can actually see them burning during landing. It is the burst of white gas coming from the side of rocket. Building an RCS is difficult. But if the team
+has prior experience with cold gas propulsion, their job becomes significantly easier. Because you need optimal nozzles for an RCS, and as mentioned before
```

Comment:
No need to specify, 'as mentioned before'. Expect readers to remember what you have written.